manifests/init.pp

```diff
@@ -128,7 +128,7 @@
 #
 # $repo_auth::                  Boolean to determine whether repos managed by
 #                               pulp will require authentication. Defaults
-#                               to true
+#                               to false
```

Comment:
We've started removing the defaults from documentation because they tend to do out of date (like you see here). Would you mind removing it?